src/hooks/useCalculateSwapPairs.ts

```diff
@@ -99,11 +99,11 @@ export function useCalculateSwapPairs(): (token?: Token) => SwapData[] {
   }, [chainId])
 
   return useCallback(
-    function calculateSwapPairs(token?: Token): SwapData[] {
+    function calculateSwapPairs(token?: BasicToken): SwapData[] {
       if (token == null || chainId == null || tokens === null) return []
```

Comment:
For the sake of keeping the scope of PR from pulling out TOKENS_MAP going to leave as is. It would be better to discuss conventions as a team and make one PR to replace everywhere.